app/components/NotificationBar.css

```diff
@@ -0,0 +1,19 @@
+.notificationBar {
+	padding: 10px 0px 10px 0px;
+}
+.notificationBar div {      
+	position:relative;
+    display:inline-block;
+	margin-left: 20%;	
+}
+.notificationBar div span span {
+    position: absolute;
+    right:-20px;
+    top:10px;
+    background: #b43430;
+    color: white;
+    text-align: center;
+    padding: 0px 5px 0px 5px;
+    font-size:12px;    
+    font-weight: bold;
```

Comment:
Let's try to get the borders slightly rounded, as similar as possible to the website. We can literally just copy and paste the CSS here if you want.